### DISCUSSION TITLE
Errors 401+403

### DISCUSSION THREAD

**Discussion by Smoulderingmel** [JeesusKrisostoomus/Twitch-Drops-Farmer#1] [Q&A] [answered]
How to fix - Exception: The remote server returned an error: (401) and step 11 on screenshot?
![GAXXSq5WUs2iR](https://github.com/JeesusKrisostoomus/Twitch-Drops-Farmer/assets/63418894/e36147e5-5bdf-46a6-a643-a967bd6fb88e)

**Comment by JeesusKrisostoomus** [comment, marked as answer]
Hello. Are you sure the Client ID and Client Secret you provided are correct?
I tried and it worked correctly.
Also were you able to access twitch normally manually?
Were you using any proxies and/or vpn-s?

**Reply by Smoulderingmel** [reply]
My mistake. I didn't see the "Client Secret" button on the console page. Thanks for the help.
